### PR TITLE
Added SurveyorHome Page and changed results controller to RestController

### PR DESCRIPTION
**Pull request by Dez1999**
Continuing to work on making surveyorHome Page dynamic

### CODE REVIEW

**Review by OmarAgamy9 — CHANGES_REQUESTED** [review]
I am confused as to what's the use of surveyorHome

**Comment by hassanidea** [comment]
Hey desmond, can you please make the connection between the surveyorHome and createSurvey.

**Comment by Dez1999** [comment]
@OmarAgamy9 SurveyorHome would allow the surveyor to have a general home page with the different surveys created. Let me know if there's anything that should be changed @hassanidea sounds good, just added it

**Review by hassanidea — APPROVED** [review]
Looks good to me now

File: src/main/resources/templates/surveyorHome.html [hunk 1]
```diff
@@ -0,0 +1,37 @@
...
+    <link href="css/surveyorHome.css" rel="stylesheet" type="text/css"/>
+</head>
+<body>
+<div style="justify-content: center;text-align:center;">
+    <h1>Surveys</h1>
+    <h2>Created Surveys shown below</h2>
+    <div style="border: thin solid black; border-radius: 10px; padding: 20px; display: inline-block">
+        <h4>All Surveys</h4>
+        <table>
+            <script>
```
Comment (keving3ng) [inline src/main/resources/templates/surveyorHome.html:15]: scripts should be placed outside of html but its fine for now

**Review by keving3ng — APPROVED** [review]

**Review by TaherShabaan99 — COMMENTED** [review]
I believe it needs more work to be done!

**Review by OmarAgamy9 — APPROVED** [review]
Looks good now

**Pull request merged**